openedx/features/edly/utils.py

```diff
@@ -108,3 +108,28 @@ def get_enabled_organizations(request):
         return []
 
     return [studio_site_edx_organization]
+
+
+def create_edly_user_profile(request, user):
+    """
+    Create edly user profile for User.
```

Comment:
Update docstring according to guidelines
https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python-guidelines.html#docstrings